codbex-payment-order-ext/generate/SalesOrderPayment/api/GenerateSalesOrderPaymentService.ts

```diff
@@ -1,4 +1,70 @@
+import { SalesOrderRepository as SalesOrderDao } from "../../../../codbex-orders/gen/codbex-orders/dao/SalesOrder/SalesOrderRepository";
+import { CustomerPaymentRepository as CustomerPaymentDao } from "../../../../codbex-payments/gen/codbex-payments/dao/CustomerPayment/CustomerPaymentRepository";
+import { SalesOrderPaymentRepository as SalesOrderPaymentDao } from "../../../../codbex-orders/gen/codbex-orders/dao/SalesOrder/SalesOrderPaymentRepository";
+
 import { Controller, Get } from "sdk/http";
 
 @Controller
-class GenerateSalesOrderPaymentService { }
\ No newline at end of file
+class GenerateSalesOrderPaymentService {
+
+    private readonly salesOrderDao;
+    private readonly customerPaymentDao;
+    private readonly salesOrderPaymentDao
+
+    constructor() {
+        this.salesOrderDao = new SalesOrderDao();
+        this.customerPaymentDao = new CustomerPaymentDao();
+        this.salesOrderPaymentDao = new SalesOrderPaymentDao();
+    }
+
+    @Get("/salesOrderData/:customerPaymentId")
+    public salesOrderData(_: any, ctx: any) {
+        const customerPaymentId = ctx.pathParameters.customerPaymentId;
+
+        let customerPayment = this.customerPaymentDao.findById(customerPaymentId);
```

Comment:
Please always use `const` modifier instead of `let` if the value is not going to be assigned to something else later in the code.

Reply:
I understood and fix that